src/main/java/net/pms/configuration/FormatConfiguration.java

```diff
@@ -115,6 +119,7 @@
 	public static final String RAW = "raw";
 	public static final String REALAUDIO_14_4 = "ra14.4";
 	public static final String REALAUDIO_28_8 = "ra28.8";
+	public static final String RGB = "rgb";
```

Comment:
Can't you just add a description above this constant that clarifies what it's used for? For example:
```java
/** Used as a "video codec" when sequences of raw, uncompressed rgb or rgba is used as a video stream in AVI files */
```